pydis_core/utils/paste_service.py

```diff
@@ -0,0 +1,95 @@
+from aiohttp import ClientConnectorError, ClientSession
+
+from pydis_core.utils import logging
+
+log = logging.get_logger(__name__)
+
+FAILED_REQUEST_ATTEMPTS = 3
+MAX_PASTE_SIZE = 256 * 1024  # 256kB
```

Comment:
Not critical, but I'd go for 128. Closer to the old limit and still a power of 2 :D

Reply:
Updated in the [force push](https://github.com/python-discord/bot-core/compare/506329a56f2642af5ff7eed53a5ced1838fe4336..801e1e5151cde2d01024fc025b8b688558720070)